Allow external roles to be installed to galaxy_roles

akofink opened the pull request:
FYI: `ansible-galaxy` installs roles to the first role in your `roles_path`.

To test, try something like this:

```sh
ansible-galaxy install bennojoy.mysql
```

ehelms commented on ansible.cfg:
```diff
@@ -2,7 +2,7 @@
 nocows = 1
 inventory = $PWD/playbooks/inventory/vagrant.py
-roles_path = $PWD/playbooks/roles
+roles_path = $PWD/galaxy_roles:$PWD/playbooks/roles
```
I was thinking of putting this side-by-side for parity: `playbooks/galaxy_roles`

ekohl commented:
Would it be useful if this was added to the readme?

akofink commented:
I updated the Readme and also added a `requirements.yml.example` and ignored `requirements.yml` for use with `ansible-galaxy install -r requirements.yml`.

ehelms commented:
Thanks @akofink
